server/src/infra/repositories/media.repository.ts

```diff
@@ -23,11 +25,28 @@ export class MediaRepository implements IMediaRepository {
   }
 
   async resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void> {
-    await sharp(input, { failOn: 'none' })
-      .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
-      .rotate()
-      .toFormat(options.format)
-      .toFile(output);
+    if (typeof input === 'string' && unsupportedSharpTypes.has(mimeTypes.lookup(input))) {
+      this.logger.log(`Sharp doesn't support ${mimeTypes.lookup(input)} type, using imagemagick instead`);
+      return new Promise((resolve, reject) => {
+        im.convert(
+          [input, '-quality', 80, '-thumbnail', `${options.size}x${options.size}^\>`, output],
```

Comment:
Should we enclose a geometry within quotation marks as [recommended](https://imagemagick.org/script/command-line-processing.php#geometry) by ImageMagick?

> In all the examples above and below, we have enclosed the geometry arguments within quotation marks. Doing so is optional in many cases, but not always. We must enclose the geometry specifications in quotation marks when using < or > to prevent these characters from being interpreted by the shell as file redirection. On Windows systems, the carat ^ needs to be within quotes, else it is ignored. To be safe, one should probably maintain a habit of enclosing all geometry arguments in quotes, as we have here.

Reply:
I think their docs refer to CLI usage from a shell, in our case, we pass by `childproc.spawn` that takes a list of arguments, so it's more like the `\` character was redundant.